packages/frontend/pages/vault/[vid].tsx

```diff
@@ -342,7 +343,13 @@ const Component: React.FC = () => {
     setAction(percent > existingCollatPercent ? VaultAction.ADD_COLLATERAL : VaultAction.REMOVE_COLLATERAL)
     setCollatPercent(percent)
     const debt = await getDebtAmount(vault.shortAmount)
-    const newCollat = new BigNumber(percent).times(debt).div(100)
+    let lpCollatPercent = 0
+    // If NFT is deposited minus don't include NFT's collat % in total collat %
```

Comment:
deposited minus?

Reply:
wdym don't include in total collat % ?

Reply:
That's a bad comment sorry. Updated it now